### ISSUE TITLE
Support for TRF7970A - over Embedded Linux

### ISSUE THREAD

**Issue by GanesanGuru** [linux-nfc/neard#23]
Hi fellas,

Whether this stack provided support for NFC - TRF7970A [module](https://www.ti.com/tool/DLP-7970ABP)? 
There is a Application [Note](https://www.ti.com/lit/an/sloa210b/sloa210b.pdf?ts=1708338869007&ref_url=https%253A%252F%252Fwww.google.com%252F) from TI sttaing the use of custom neard lib for AM33x alone.

We are trying to incorporate for the support for TRF7970A module in embedded linux - built using Buildroot for Microchip vendor. 

Kindly let me know your inference.

**Comment by mgreeraz** [comment]
Yes, if you have the trf7970a working in your linux kernel, neard should work for you. Did that answer your question?
